deploy/charts/rook-ceph-cluster/templates/cephobjectstore.yaml

```diff
@@ -20,6 +20,8 @@ metadata:
 {{- end }}
 {{- if $objectstore.storageClass.annotations }}
   annotations:
+    "helm.sh/hook": post-install,pre-upgrade
```

Comment:
No need of this for object store sc

Reply:
> why? is it not not imuatable?

It is but we are not changing parameters of object store sc right now
To fix the upgrade issue, only csi storageclasses need this annotations